SPECS/libmodulemd/libmodulemd.spec

```diff
@@ -1,15 +1,14 @@
 %bcond_with docs
 Summary:        Module manipulating metadata files
 Name:           libmodulemd
-Version:        2.13.0
-Release:        2%{?dist}
+Version:        2.15.0
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Applications/System
 URL:            https://github.com/fedora-modularity/libmodulemd
 Source0:        https://github.com/fedora-modularity/libmodulemd/releases/download/%{version}/modulemd-%{version}.tar.xz
-Patch1:         test_import_headers_timeout.patch
```

Comment:
This patch was added in https://github.com/microsoft/CBL-Mariner/commit/433135024b3b46d6ee9c3342e62d0fbef589a4e9#diff-e1481c9086f54a51996148760bf4690135a008fab4737e899b98a019c5e7f3d6. I believe this patch increases the timeout of a ptest. I believe I can remove it as I was able to pass all package tests in buddy build, despite removing the patch